assets/js/buttons-examples.js

```diff
@@ -21,6 +21,12 @@ $(document).ready(function () {
       classes: "basic-btn",
       children: [],
     },
+    {
```

Comment:
Please don't add it as a new type. Add it in the children of Animated in the correct alphabetical order